src/components/settings/SettingsLMSTab/ExistingLMSCardDeck.jsx

```diff
@@ -0,0 +1,153 @@
+import React, { useState } from 'react';
+import isEmpty from 'lodash/isEmpty';
+import PropTypes from 'prop-types';
+import {
+  Badge, Card, CardGrid, Dropdown, Icon, IconButton, Image, useToggle,
+} from '@edx/paragon';
+import {
+  Edit, MoreVert, PlayCircleFilled, RemoveCircle,
+} from '@edx/paragon/icons';
+import { channelMapping } from '../../../utils';
+import { handleErrors } from './LMSConfigPage';
+import { SUCCESS_LABEL } from '../data/constants';
+import ConfigError from './ConfigError';
+
+const errorModalText = 'We were unable to toggle your config. Please try submitting again or contact support for help.';
+
+const ExistingLMSCardDeck = ({
+  configData,
+  editExistingConfig,
+  enterpriseCustomerUuid,
+  onClick,
+}) => {
+  const [errorIsOpen, openError, closeError] = useToggle(false);
+  const [erroredConfig, setErroredConfig] = useState();
+
+  const toggleConfig = async (id, channelType, toggle) => {
+    const configOptions = {
+      active: toggle,
+      enterprise_customer: enterpriseCustomerUuid,
+    };
+
+    let err;
+    try {
+      await channelMapping[channelType].update(configOptions, id);
+    } catch (error) {
+      err = handleErrors(error);
+    }
+    if (err) {
+      openError();
+      setErroredConfig({ id, channelType, toggle });
+    } else {
+      onClick(SUCCESS_LABEL);
+    }
+  };
+  // Map the existing config data to individual cards
+  const listItems = configData.map((config) => (
+    <Card isClickable tabIndex="0" className="p-2.5 existing-lms-card-width" key={config.channelCode + config.id}>
+      <Card.Header
+        className="lms-card-content"
+        actions={(
+          <Dropdown>
+            <Dropdown.Toggle
+              id="dropdown-toggle-with-iconbutton"
+              as={IconButton}
+              src={MoreVert}
+              iconAs={Icon}
+              variant="primary"
+              alt="Actions dropdown"
+            />
+            <Dropdown.Menu>
+              {isEmpty(config.isValid.missing) && !config.active && (
+                <div className="d-flex">
+                  <Dropdown.Item
+                    onClick={() => toggleConfig(config.id, config.channelCode, true)}
```

Comment:
This enable and disable action both show up with the "Your configuration has been submitted successfully" toast. Idk if this was supposed to do that, but if it does show a toast, I think that the wording should be different depending on the action.

Reply:
added customized wording for toggle and delete toasts! 👍